app/helpers/tasks_helper.rb

```diff
@@ -0,0 +1,46 @@
+module TasksHelper
+  TASKS_LIMIT = 5
+
+  def all_assigned_tasks
+    @assigned_tasks ||= begin
+      tasks = current_user.cards
+        # Using Arel.sql to sort the records by due_date with null due_date records last
+        .order(Arel.sql('due_date IS NULL, due_date ASC'))
+
+      if params[:project_id]
+        return tasks.select { |card| card.project.id == current_project.id }
+      end
+
+      tasks
+    end
+  end
+
+  def assigned_tasks
```

Comment:
what about `assigned_tasks_for_widget` or something to make it clear that this is a limited list?